File: Leetcode/136-Single-Number/SingleNumber.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        if (nums.empty()) {
            return 0;
        }
        int res = 0;
        for(auto i=nums.begin();i<nums.end();i++) {
            res = res^*i;
        }
        return res;
    }
};

File: Leetcode/136-Single-Number/SingleNumber.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if nums == None or len(nums) == 0:
            return 0
        res = 0
        for i in nums:
            res ^= i
        return res

